apps/condo/pages/ticket/index.tsx

```diff
@@ -172,6 +184,43 @@ export const TicketsPageContent = ({
     } = Ticket.useCount({ where: baseTicketsQuery })
     const { count: ticketsWithFiltersCount } = Ticket.useCount({ where: searchTicketsQuery })
 
+    const { useFlag, updateContext } = useFeatureFlags()
+    const { user } = useAuth()
+    const isTicketImportFeatureEnabled = useFlag(TICKET_IMPORT)
+    const [columns, ticketNormalizer, ticketValidator, ticketCreator] = useImporterFunctions()
+
+    const TicketImportButton = useMemo(() => {
+        return isTicketImportFeatureEnabled && (
+            <ImportWrapper
+                accessCheck={isTicketImportFeatureEnabled}
+                domainTranslate={TicketReadingObjectsNameManyGenitiveMessage}
+                columns={columns}
+                objectsName={TicketsMessage}
+                onFinish={undefined}
+                rowValidator={ticketValidator}
+                rowNormalizer={ticketNormalizer}
+                objectCreator={ticketCreator}
+                exampleTemplateLink='/ticket-import-example.xlsx'
+                maxTableLength={
+                    hasFeature('bigger_limit_for_import')
+                        ? EXTENDED_RECORDS_LIMIT_FOR_IMPORT
+                        : DEFAULT_RECORDS_LIMIT_FOR_IMPORT
+                }
+
+            >
+                <Button
+                    type='sberPrimary'
+                    icon={<DiffOutlined/>}
+                    secondary
+                />
+            </ImportWrapper>
+        )
+    }, [TicketReadingObjectsNameManyGenitiveMessage, TicketsMessage, columns, isTicketImportFeatureEnabled, ticketCreator, ticketNormalizer, ticketValidator])
+
+    useEffect(() => {
+        updateContext({ isSupport: user.isSupport || user.isAdmin })
+    }, [updateContext, user.isAdmin, user.isSupport])
```

Comment:
Context is object, so does `useEffect` work correctly here? No re-rendering? Maybe something like `useDeepCompareEffect` needed?

Reply:
there is no difference